src/Bindings/ManualBindings.cpp

```diff
@@ -3684,6 +3684,48 @@ static int tolua_cEntity_GetSpeed(lua_State * tolua_S)
 
 
 
+static int tolua_cColor_Set(lua_State * tolua_S)
+{
+	cLuaState L(tolua_S);
+
+	if (!L.CheckParamNumber(2, 4) || !L.CheckParamSelf("cColor"))
+	{
+		return 0;
+	}
+	cColor * self = reinterpret_cast<cColor *>(tolua_tousertype(tolua_S, 1, nullptr));
```

Comment:
Your suspicion is correct. So why not add the `cColor` to ToLua? It would then gain all the partial getters and setters, you wouldn't need to export `SetColor` manually, and `GetColor` could use `GetStackValues` for a `cColor *` value.